gdk4/sys/src/lib.rs

```diff
@@ -4209,6 +4218,8 @@ extern "C" {
         minor: *mut c_int,
     );
     pub fn gdk_gl_context_is_legacy(context: *mut GdkGLContext) -> gboolean;
+    pub fn gdk_gl_context_is_shared(self_: *mut GdkGLContext, other: *mut GdkGLContext)
```

Comment:
Needs a since annotation

Reply:
crap, that went unnoticed in 4.3.2.. https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3790
How do I override that in gir now? :)

Reply:
You can override the version needed for the function & add a comment to drop it next gir files bump

Reply:
done